src/Module.ps1

```diff
@@ -39,6 +79,45 @@ $script:SafeCommands['Set-DynamicParameterVariable'] = $ExecutionContext.Session
     'New-PesterContainer'
     'New-PesterConfiguration'
 
+    # assert
+    'Assert-False'
+    'Assert-True'
+    'Assert-Falsy'
```

Comment:
Because we are aliasing those `Assert-*` functions to `Should-*`, that is the least hacky way to publish command with non-approved verb. There are probably other ways around, but this one is the official way. 

But frankly I did not try if you can export alias of non-exported function.

Reply:
I was thinking more about why we use Assert-* as the function name in the first place. Noisy duplicate commands/aliases, side effects on help (`Get-Help Should-Be` showing Name `Assert-Be`) and docs-generation etc.

Forgot that Assert is an approved verb so I get it, thanks.

> But frankly I did not try if you can export alias of non-exported function.

For good reason, it won't work 🙂